Style.css

```diff
@@ -462,3 +451,197 @@ body {
   color: #b3bac5;
   opacity: 80%;
 }
+
+.buttonC:active {
+  background-color: #2230d2;
+  color: #fff;
+}
+
+.buttonMenu { display: none; }
+
+.mobile2 {
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+}
+
+.buttonC:hover 
+{

```

Comment:
I believe the issue is here 👇🏽 

![Screenshot 2023-05-07 at 01 15 27](https://user-images.githubusercontent.com/45739963/236659109-4f0d90eb-106e-44cf-98fb-67e5a5c0019f.png)

Should be fixed if you replace these two lines by this :

```
.buttonC:hover {
```